docs/open-api/openapi.yaml

```diff
@@ -477,7 +477,21 @@ components:
           - "TOPIC"
           - "ROLE"
           - "METALAKE"
-
+    metadataObjectTypeWithOwner:
+      name: metadataObjectType
+      in: path
+      description: The type of the metadata object
+      required: true
+      schema:
+        type: string
+        enum:
+          - "CATALOG"
+          - "SCHEMA"
+          - "TABLE"
+          - "FILESET"
+          - "TOPIC"
+          - "ROLE"
+          - "METALAKE"
```

Comment:
plz move`METALAKE` in front of `CATALOG`, as `METALAKE` has a larger scope

Reply:
Ok

Reply:
Do I have to move original metadataObjectType's "METALAKE" as well?

Reply:
You can do either; it’s up to you.